scripts/pre_push_hook.py

```diff
@@ -181,7 +196,7 @@ def git_diff_name_status(left, right, diff_filter=''):
     git_cmd = ['git', 'diff', '--name-status']
     if diff_filter:
         git_cmd.append('--diff-filter={}'.format(diff_filter))
-    git_cmd.extend([left.encode('utf-8'), right.encode('utf-8')])
+    git_cmd.extend([left.encode('utf-8'), right.encode('utf-8')])  # type: ignore[list-item]
```

Comment:
....

Reply:
No, I think it should be all strings or all bytes -- thanks for flagging. Do we typically use str or bytes for other cmds? We should do the same thing for this command.

(After you've changed the code, try invoking it manually to confirm that it works.)

Reply:
> Do we typically use str or bytes for other cmds?

we use strings for other cmds. So, `.encode('utf-8')` is removed.

Reply:
> try invoking it manually to confirm that it works 

Everything is working fine!